source/fundamentals/crud/write-operations/delete.txt

```diff
@@ -0,0 +1,106 @@
+=================
+Delete a Document
```

Comment:
Should we mention [findOneAndDelete](https://mongodb.github.io/mongo-java-driver/4.3/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#findOneAndDelete(org.bson.conversions.Bson,com.mongodb.client.model.FindOneAndDeleteOptions))?